src/app/legacy/containers/Header/index.jsx

```diff
@@ -76,6 +77,7 @@ const HeaderContainer = ({ propsForTopBarOJComponent }) => {
 
   if (scriptLink) {
     switch (true) {
+      case ['zhongwen', 'serbian'].includes(service) && pageType === LIVE_PAGE:
```

Comment:
We could technically simplify this even more.

We only get into this code if the service has a script link (serbian, uzbek, ukchina + zhongwen), and we know that script switching does not work for any of these services yet

```suggestion
      pageType === LIVE_PAGE:
```

Reply:
Good point, updated